### PR TITLE
android: Add screen orientation presets to layout settings

### LINKED ISSUE #194: add Orientation option
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What feature are you suggesting?

Orientation option
Screenshot_2024-05-17-16-12-23-657_org yuzu yuzu_emu

### Why would this feature be useful?

u don't have to rotate your phone everytime because its pretty annoying.

### PR DESCRIPTION
**Pull request by ghost**
Updated version of https://github.com/Lime3DS/Lime3DS/pull/270 for the current codebase

This PR adds a new popup strings menu to the "Layout" section of Settings with locked screen orientations

Closes https://github.com/Lime3DS/Lime3DS/issues/194 and https://github.com/Lime3DS/Lime3DS/issues/95 (IDK why there were 2 duplicates)

### CODE REVIEW

File: src/android/app/src/main/res/values/arrays.xml [hunk 1]
```diff
@@ -530,4 +530,32 @@
...
+        <item>0</item>
+        <item>1</item>
+    </integer-array>
+
+    <string-array name="orientationOptions">
+        <item>Auto (Sensor)</item>
+        <item>Landscape</item>
+        <item>Landscape (Flipped)</item>
+        <item>Portrait</item>
+        <item>Portrait (Flipped)</item>
```
Comment (OpenSauce04) [inline src/android/app/src/main/res/values/arrays.xml]: These need to be in `strings.xml`

**Review by OpenSauce04 — CHANGES_REQUESTED** [review]

File: src/android/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -361,6 +361,12 @@
     <string name="emulation_control_dpad_slide_enable">D-Pad Sliding</string>
     <string name="emulation_open_settings">Open Settings</string>
     <string name="emulation_open_cheats">Open Cheats</string>
+    <string name="emultaion_screen_orientation_option">Screen Orientation Option</string>
```
Comment (OpenSauce04) [inline src/android/app/src/main/res/values/strings.xml]: This string doesn't even work because the name is misspelled, did you test this code?

Comment (ghost) [reply src/android/app/src/main/res/values/strings.xml]: Yeah, before screwing it up when moving it to strings.xml

And not noticing 

Classic

File: src/android/app/src/main/java/io/github/lime3ds/android/display/ScreenAdjustmentUtil.kt [hunk 3]
```diff
@@ -56,4 +61,17 @@ class ScreenAdjustmentUtil(
         NativeLibrary.reloadSettings()
         NativeLibrary.updateFramebuffer(NativeLibrary.isPortraitMode)
     }
+
+    fun changeActivityOrientation(orientationOption: Int) {
+        if (currentOrientation == orientationOption) {
+            return
+        }
```
Comment (OpenSauce04) [inline src/android/app/src/main/java/io/github/lime3ds/android/display/ScreenAdjustmentUtil.kt]: Why is this necessary? What happens if you set the orientation to the current one which we are trying to avoid?

**Review by OpenSauce04 — CHANGES_REQUESTED** [review]

**Review by OpenSauce04 — APPROVED** [review]
